libs/sdk-backend-tiger/api/sdk-backend-tiger.api.md

```diff
@@ -13,8 +13,6 @@ import { AxiosRequestConfig } from 'axios';
 import { DataSourceParameter } from '@gooddata/api-client-tiger';
 import { DeclarativeAnalytics } from '@gooddata/api-client-tiger';
 import { DeclarativeModel } from '@gooddata/api-client-tiger';
-import { DeclarativePdm } from '@gooddata/api-client-tiger';
```

Comment:
Removed thing is no longer part of backend API, I had to remove it for the file to compile. It seems when PDM was removed, they removed API as well.